contrib/gitian-keys/README.md

```diff
@@ -1,9 +1,10 @@
 ## PGP keys of Gitian builders and Developers
 
-The keys.txt contains the public keys of Gitian builders and active developers.
+The file *keys.txt* contains fingerprints of the public keys of Gitian builders
```

Comment:
I'd suggest to use `keys.txt` (e.g. backticks) if you want to emphasize the filename, this is a convention used throughout the docs.

Reply:
Updated keys.txt markup (I think).